entries/b/bokio.se.json

```
@@ -0,0 +1,20 @@
+{
+  "Bokio Sverige": {
+    "domain": "bokio.se",
+    "url": "https://www.bokio.se",
+    "img": "bokio.svg",
+    "tfa": [
+      "totp",
+      "custom-software"
+    ],
+    "notes": "Custom software BankID used for users using built-in banking features",
```

2FA needs to be presented upon login. If it's just for certain actions or integration with third party APIs then it shouldn't be listed.
```suggestion
      "totp"
    ],
```

---

This one is a bit special where all the banking users are forced to login with BankID upon login. Non banking users can use `totp`. So both types of 2FA are presented upon login but based on the type of user. Any suggestion on how to write that hear? @Carlgo11

---

Okay let's leave BankID in then. Just specify it in the custom-software element:
```suggestion
      "totp",
      "custom-software"
    ],
    "custom-software": [
      "BankID"
    ],
    "notes": "BankID only available for users using banking features",
```

Out of curiosity, do you use your own auth tokens for BankID or are you using the bank's?

---

(I also work at Bokio.)

With [BankID](https://en.wikipedia.org/wiki/BankID), the user authentication process including verification of enrolled two-factor devices is handled by the bank (or actually by BankID, which is co-owned by the big banks in Sweden). Bokio integrates with BankId as an identity provider. Does that answer your question?

---

@miniyou @erzki I just wanted to know if you're making the BankID calls from the bank's API keys or your own? i.e. who pays for the auth attempts.

When I built a banking dashboard a few years back I used Swedbanks app API and also send the auth requests using their API resulting in Swedbank showing up as the company on the BankID screen.

---

I see :) Bokio integrates with BankId directly and `Bokio AB` is shown in the BankId authentication screen.